docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -466,7 +396,7 @@ To build TF-M and create an Mbed OS pull request:
     python3 build_tfm.py -m <new target> -t <toolchain> --commit
```

Comment:
This is a linux only command ?

Maybe mbed cli2 could be used ? mbedtools tfm -m xxx ?

Reply:
It runs on Windows and macOS too, as long as python and dependencies are installed. Mbed CLI 2 support for PSA/TF-M targets and repositories will be added, at which point we'll update the documentation.